examples: Update docs to redirect users to the appropriate release (#265)
ni/measurementlink-python: examples/README.md

## Example Measurements

These are example measurements for MeasurementLink 2023 Q3 or later.

If you are using a previous version of MeasurementLink, download the appropriate examples:

> - MeasurementLink 2023 Q1: [measurementlink-python-examples-1.0.1.zip](https://github.com/ni/measurementlink-python/releases/download/1.0.1/measurementlink-python-examples-1.0.1.zip)

Reviewer: It looks like you retroactively created the example assets? I can do that for LabVIEW too.

## Example Measurements

### Downloading the Example Measurements

> Example measurements are available in the `examples` directory of the `measurementlink-python` Git repository. (That's this directory.) You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.

Reviewer: ```suggestion
This directory contains example measurements. You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.
```

Feel free to disagree with my suggestion. I think the `That's this directory` statement is awkward so I'm trying to get rid of it.

Author: `examples/README.md` now says "These are example measurements for MeasurementLink 2023 Q3 or later," which is correct no matter whether you are looking at it on GitHub, a local Git clone, or a downloaded copy of `measurementlink-python-examples-1.1.0-dev0.zip`.
